pkg/sql/logictest/testdata/logic_test/views

```diff
@@ -140,6 +140,27 @@ CREATE DATABASE test2
 statement ok
 SET DATABASE = test2
 
+statement ok
+CREATE VIEW t1 AS SELECT ARRAY[]:::STRING[];
+CREATE VIEW t2 AS SELECT ARRAY[1,2,3]:::INT[];
+
+
+query TT colnames
+SHOW CREATE VIEW t1
+----
+table_name  create_statement
+t1          CREATE VIEW public.t1 (
+              "array"
+            ) AS SELECT ARRAY[]::STRING[]:::STRING[]
+
+query TT colnames
+SHOW CREATE VIEW t2
+----
+table_name  create_statement
+t2          CREATE VIEW public.t2 (
+              "array"
+            ) AS SELECT ARRAY[1:::INT8, 2:::INT8, 3:::INT8]::INT8[]:::INT8[]
```

Comment:
it looks like this still has an extra `:::INT8[]` type annotation at the end. (and same for `t1` above this)